map_list.schema.yaml

```diff
@@ -92,6 +92,8 @@ additionalProperties:
               additionalProperties: false
               required:
                 - poly
+          maxPlayersPerStartbox:
+            type: integer
```

Comment:
Could you also add constraint on the values https://json-schema.org/understanding-json-schema/reference/numeric.html#range? I believe we want it to be `1 <= maxPlayersPerStartbox <= 8`.

Reply:
Done.